knimin/lib/tests/test_vioscreen.py

```diff
@@ -0,0 +1,327 @@
+from unittest import TestCase, main
+from datetime import datetime
+from os.path import join, dirname, realpath
+from six import StringIO
+import json
+
+from knimin.lib.vioscreen import VioscreenHandler
+
+
+class TestVioscreenHandler(TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Why `cls` instead of `self`?

Reply:
I looked up setUpClass() which was necessary for my testing because it only sets up once for all of the testing instead of before each test. It's listed as a classmethod (which I haven't exposure to before) and in looking up classmethod, it looks like each use of it has cls as the subject. https://stackoverflow.com/questions/12179271/meaning-of-classmethod-and-staticmethod-for-beginner

Reply:
Oh, I see... ya, okay